js/simpletextbox/Index.svelte

```diff
@@ -0,0 +1,167 @@
+<svelte:options accessors={true} />
+
+<script lang="ts">
+	import type { Gradio, SelectData } from "@gradio/utils";
+	import { BlockTitle } from "@gradio/atoms";
+	import { Block } from "@gradio/atoms";
+	import { StatusTracker } from "@gradio/statustracker";
+	import type { LoadingStatus } from "@gradio/statustracker";
+	import { tick } from "svelte";
+
+	export let gradio: Gradio<{
+		change: string;
+		submit: never;
+		blur: never;
+		select: SelectData;
+		input: never;
+		focus: never;
+	}>;
+	export let label = "Textbox";
+	export let elem_id = "";
+	export let elem_classes: string[] = [];
+	export let visible = true;
+	export let value = "";
+	export let placeholder = "";
+	export let show_label: boolean;
+	export let scale: number | null = null;
+	export let min_width: number | undefined = undefined;
+	export let loading_status: LoadingStatus | undefined = undefined;
+	export let value_is_output = false;
+	export let mode: "static" | "interactive";
+	export let rtl = false;
+
+
+	let el: HTMLTextAreaElement | HTMLInputElement;
+	const container = true;
+
+	function handle_change(): void {
+		gradio.dispatch("change", value);
+		if (!value_is_output) {
+			gradio.dispatch("input");
+		}
+	}
+
+	function handle_select(event: Event): void {
+		const target: HTMLTextAreaElement | HTMLInputElement = event.target as
+			| HTMLTextAreaElement
+			| HTMLInputElement;
+		const text = target.value;
+		const index: [number, number] = [
+			target.selectionStart as number,
+			target.selectionEnd as number,
+		];
+		gradio.dispatch("select", { value: text.substring(...index), index: index });
+	}
```

Comment:
i think we could definitely get rid of the select event. I don't think it used very much and this adds a lot of noise to the file.